app/models/credit_card.rb

```diff
@@ -1,4 +1,4 @@
 class CreditCard < ApplicationRecord
-  validates :first_name, :last_name, :card_number, :expiration_date, presence: true 
+  validates :first_name, :last_name, :token, :expiration_date, presence: true
```

Comment:
expiration_date doesn't make a lot of sense to store. generally storing the year and month is common.